openstack/imageservice/v2/results.go

```diff
@@ -0,0 +1,349 @@
+package v2
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	
+	"github.com/rackspace/gophercloud"
+	//"github.com/rackspace/gophercloud/pagination"
+)
+
+// does not include the literal image data; just metadata.
+// returned by listing images, and by fetching a specific image.
+type Image struct {
+	Id string
+	
+	Name string
+	
+	Status ImageStatus
+	
+	Tags []string
+	
+	ContainerFormat *string
```

Comment:
If the empty string is a valid option for `ContainerFormat`, then the `*string` is necessary. Otherwise, let's use `string` and only set that field if `ContainerFormat` is non-empty.